app/controllers/content_controller.rb

```diff
@@ -14,7 +14,9 @@ def get_content
     response = FindContent.call(content_params)
     ContentItemsPresenter.new(
       response[:results],
-      DateRange.new(content_params[:date_range])
+        DateRange.new(content_params[:date_range]),
```

Comment:
The indentation seems to be broken here.
We have increased the number of parameters that the Presenter is receiving. I am wondering about passing only two: `response` and `content_params`? Other can be inferred from these two parameters.